src/sdk/conference/client.js

```diff
@@ -416,13 +416,20 @@ export const ConferenceClient = function(config, signalingImpl) {
             }
           }
         }
-        if (typeof QuicTransport === 'function' && token.webTransportUrl) {
+        if (typeof WebTransport === 'function' && token.webTransportUrl) {
           quicTransportChannel = new QuicConnection(
               token.webTransportUrl, resp.webTransportToken,
               createSignalingForChannel(), config.webTransportConfiguration);
         }
-        resolve(new ConferenceInfo(resp.room.id, Array.from(participants
-            .values()), Array.from(remoteStreams.values()), me));
+        const conferenceInfo=new ConferenceInfo(resp.room.id, Array.from(participants
```

Comment:
spacing.

Reply:
It looks like you were looking at an old version. Spacing issues should be detected by linting during CI. It has been fixed in the latest version.